server.js

```diff
@@ -166,6 +166,26 @@ async.map(
   }
 );
 
+// app.post('/', function(req, res) {
```

Comment:
@jace1084 You need to put this code into a new file in the routes folder and then require it in the server.js. You can check out deals.js in the routes folder for example. We break the api routes into separate files.